### PR TITLE
[BUGFIX] Prevent ECAM auto page switching when page is selected

### PR DESCRIPTION
**Pull request by lukecologne**
**Summary of Changes**
* Fixes issue where automatic ECAM page selection would override the manually selected page.

**Additional context**
* ~~Maybe a problem: A page triggered by a failure will not override the selected page, I don't know if this is expected behaviour or not~~
* ~~Problem: When Engine mode selector is not in normal, the engine page would come up, but will not go away, somewhere it overrides the currently selected page.~~ Should be fixed now

### CODE REVIEW

**Comment by lukecologne** [comment]
Ok, the switching happens in the XML template for the Engine mode selector. We need to override it

**Comment by lukecologne** [comment]
Done. I'm not really experienced with this XML switch stuff, so if I should change something, let me know.

**Comment by AdenFlorian** [comment]
> Maybe a problem: A page triggered by a failure will not override the selected page, I don't know if this is expected behaviour or not

Please find out if it is or isn't. I recommend asking in #ecam

**Comment by lukecologne** [comment]
According to the FCOM, it should replace the selected page
grafik
I'll come up with a solution

**Comment by lukecologne** [comment]
Alright, should now work as expected

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/A320_Neo_EICAS.js [hunk 1]
```diff
@@ -244,7 +245,7 @@ class A320_Neo_EICAS extends Airliners.BaseEICAS {
             }
         } else if ((ToPowerSet || !Simplane.getIsGrounded()) && !crzCond && this.minPageIndexWhenUnselected <= 2) {
             this.pageNameWhenUnselected = "ENG";
-        } else if (crzCond && !(isGearExtended && altitude < 16000) && this.minPageIndexWhenUnselected <= 3) {
```
Comment (AdenFlorian) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/A320_Neo_EICAS.js:247]: why was this removed and not the other instances of it?

**Comment by AdenFlorian** [comment]
talked with @lukecologne , I think I get it better now, they will also revert the change that made the failure pages change the selected button.

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/A320_Neo_EICAS.js (hunk 1 above)
Comment (lukecologne) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/A320_Neo_EICAS.js:247]: Should I revert this as well?

**Comment by lukecologne** [comment]
Alright, this should do it. There is an issue with the XMLVAR_ECAM_CURRENT_PAGE rapidly switching values after the user has cleared all memos with the CLR button, which causes some weird flickering on the ECAM button light. This has to be an issue somewhere else though, because I don't set the Simvar in my code.

**Comment by camilo625** [comment]
Works great!

**Comment by camilo625** [comment]
One cuestion, if you are seeing the automatic page, and switch ignition ON, you will see the engine page, but if you turn ignition off it will stay on the engine page (basically it switches your automatic page), its that normal?

**Comment by lukecologne** [comment]
Yes: 
grafik

**Comment by lukecologne** [comment]
See last item

**Comment by AdenFlorian** [comment]
@lukecologne can you confirm whether the rapid switching issue that you mentioned exists on master or not?

**Comment by lukecologne** [comment]
I'll rebase onto master, and see if the issue still exists

**Comment by lukecologne** [comment]
No, seems to look fine now.
